tools/params_processors/swift_debug_settings_processor.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/python3
```

Comment:
Not particular to this PR but I think the shebang is not needed, since the script is not invoked directly, and this file doesn't need to be executable.

Reply:
true, technically not needed, but I was testing it by running it directly so I wanted it.